passkey-types/src/ctap2/attestation_fmt.rs

```diff
@@ -252,7 +252,8 @@ impl AttestedCredentialData {
 
 impl AttestedCredentialData {
     /// Custom implementation rather than IntoIterator because the iterator type is complicated.
-    fn into_iter(self) -> impl Iterator<Item = u8> {
+    #[allow(clippy::should_implement_trait)]
```

Comment:
Question: what was it complaining about?

Reply:
It was saying that a public function called `into_iter` should be replaced by an implementation of `IntoIterator`, but based on the comment it seems like that would have been complicated.